instituto-brasileiro-de-informacao-em-ciencia-e-tecnologia-abnt.csl

```diff
@@ -358,37 +398,158 @@
+            </if>
+            <else>
+              <text term="no-place" form="short" text-case="capitalize-first" font-style="italic" prefix="[" suffix="]"/>
+            </else>
+          </choose>
+          <choose>
+            <if variable="publisher">
+              <text variable="publisher"/>
+            </if>
+            <else>
+              <text term="no-publisher" form="short" font-style="italic" prefix="[" suffix="]"/>
+            </else>
+          </choose>
+        </group>
+      </if>
+      <else>
+        <group delimiter=": ">
+          <text term="no-place" form="short" text-case="capitalize-first" font-style="italic" prefix="["/>
+          <text term="no-publisher" form="short" font-style="italic" suffix="]"/>
+        </group>
+      </else>
+    </choose>
+  </macro>
+  <macro name="publisher-issued">
+    <choose>
+      <if variable="publisher-place publisher" match="any">
+        <group delimiter=": ">
+          <choose>
+            <if variable="publisher-place">
+              <text variable="publisher-place"/>
+            </if>
+            <else>
+              <text term="no-place" form="short" text-case="capitalize-first" font-style="italic" prefix="[" suffix="]"/>
+            </else>
+          </choose>
+          <choose>
+            <if variable="publisher">
+              <text variable="publisher"/>
+            </if>
+            <else>
+              <text term="no-publisher" font-style="italic" prefix="[" suffix="]"/>
+            </else>
+          </choose>
+        </group>
+      </if>
+      <else>
+        <group delimiter=": ">
+          <text term="no-place" form="short" text-case="capitalize-first" font-style="italic" prefix="["/>
+          <text term="no-publisher" form="short" font-style="italic" suffix="]"/>
+        </group>
+      </else>
+    </choose>
+    <choose>
+      <if variable="issued" match="any">
+        <date variable="issued" delimiter=" " prefix=", ">
+          <date-part name="day"/>
+          <date-part name="month" form="short"/>
+          <date-part name="year"/>
+        </date>
+      </if>
+      <else>
+        <text term="no date" form="short" font-style="italic" prefix=", [" suffix="]"/>
+      </else>
+    </choose>
+  </macro>
+  <macro name="submitted">
+    <choose>
+      <if variable="submitted" match="any">
+        <group>
+          <choose>
+            <if type="bill book broadcast chapter map manuscript paper-conference report" match="none">
+              <date variable="submitted">
+                <date-part name="day" suffix=" "/>
+                <date-part name="month" form="short" suffix=" "/>
+              </date>
+            </if>
+          </choose>
+          <date variable="submitted">
+            <date-part name="year"/>
+          </date>
+        </group>
+      </if>
+      <else>
+        <text term="no date" form="short" font-style="italic" prefix="[" suffix="]"/>
```

Comment:
The vast majority of items do not have submitted dates. This doesn't seem correct to show here